selfapi.js

```diff
@@ -367,6 +374,24 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  buildIndexRoutes: function (basePath) {
+      var fullPath = normalizePath(this.path, basePath) || '/';
```

Comment:
Nit: You're indenting with 4 spaces here. Please stay consistent with the rest of this code base, which uses 2 space indents.